pairing.php

```diff
@@ -45,8 +47,28 @@ function calculate(PDO $pdo): self
     }
 
     //TODO: generate sql to add the pairing to completed rounds
-    function sql($id, $i1, $i2): string
+    function sql(PDO $pdo, string $id, item $i1, item $i2): string
     {
-        throw new Error("sql not implemented");
+        $insertSql = "INSERT INTO completed_rounds (tournament_id, item1_id, item2_id, player, winner) 
+                      VALUES (:tournament_id, :item1_id, :item2_id, :player, :winner);";
+        $insertParams = [
+            ':tournament_id' => $id,
+            ':item1_id' => $i1->id,
+            ':item2_id' => $i2->id,
+            ':player' => $this->player,
+            ':winner' => $this->winner
+        ];
+
+        $updateSql1 = $i1->getUpdateSql();
```

Comment:
getUpdateSql doesn't exist, it will also be added to the database at other times. This function should just do the sql to put the round into the rounds database.